src/pages/Home/Home.jsx

```diff
@@ -103,7 +137,7 @@ function Home()
                     </div>
 
                     <div className='active-contacts-container'>
-                        <h3>Active Contacts</h3>
+                        <h3>Active Users</h3>
                         <hr></hr>
```

Comment:
instead of writing **hr** tag this way you can do this like  cause hr tags are self-closing tags so you don't need to write it this way